python/cudf/cudf/core/dataframe.py

```diff
@@ -3955,6 +3955,12 @@ def apply(
         For more information, see the `cuDF guide to user defined functions
         <https://docs.rapids.ai/api/cudf/stable/user_guide/guide-to-udfs.html>`__.
 
+        Support for use of string data within UDFs is provided through the
+        ``strings_udf`` rapids library. Supported operations on strings include
+        the subset of functions and string methods that expect an input string
+        but do not return a string. Refer to caveats in the ``guide-to-udfs``
+        notebook referenced above.
```

Comment:
It's almost an implementation detail that the guide is written as a notebook -- ultimately the user only sees the rendered HTML page.

```suggestion
        but do not return a string. Refer to caveats in the
        UDF guide referenced above.
```

Reply:
Ditto below

Reply:
Let's link it again? Or does that become too noisy?